src/capi_frontend/inferencerequest.cpp

```diff
@@ -118,4 +139,10 @@ std::map<std::string, shape_t> InferenceRequest::getRequestShapes() const {
     }
     return result;
 }
+
+void InferenceRequest::setCompletionCallback(OVMS_InferenceResponseCompleteCallback_t callback, void* callbackData) {
```

Comment:
If we switched from Complete to Completion in public APIs, I think we should also change another public name:
OVMS_InferenceResponse**Complete**Callback_t -> OVMS_InferenceResponse**Completion**Callback_t